lib/sda/host.go

```diff
@@ -656,3 +661,27 @@ func (h *Host) checkPendingTreeMarshal(el *EntityList) {
 	}
 	h.pendingTreeLock.Unlock()
 }
+
+// Testing helpers functions
+// NewHostMock returns a freshly generated host at this address with a random
+// keypair
+func NewHostMock(s abstract.Suite, address string) *Host {
```

Comment:
Do you know how can you import theses functions from another package if they are in a *_test package ? I can't find ( I did not search too long either ). Here I need theses functions outside SDA, and I can't put it in sda_test because I don't know how to import them. I could put them in jvss_test.go but then everyone will have to repeat the same code over and over...